### ISSUE TITLE
400 when playing the auction endpoint following ReadMe

### ISSUE THREAD

**Issue by StarWindMoonCloud** [prebid/prebid-server#1335]
command and output:
curl --data '{"description":"This uses Appnexus to fetch the prebid sample ad, as seen on prebid.org.","requestPayload":{"id":"some-request-id","site":{"page":"prebid.org"},"imp":[{"id":"some-impression-id","banner":{"format":[{"w":300,"h":250},{"w":300,"h":600}]},"ext":{"appnexus":{"placementId":12883451}}}],"tmax":500}}' "http://localhost:8000/openrtb2/auction"
Invalid request: request missing required field: "id"

doc
https://github.com/prebid/prebid-server/blob/master/docs/endpoints/openrtb2/auction.md

**Comment by StarWindMoonCloud** [comment]
figured out, it should be json node of requestPayload instead of the whole json here, which is confusing though
view-source:https://raw.githubusercontent.com/prebid/prebid-server/master/endpoints/openrtb2/sample-requests/valid-whole/exemplary/prebid-test-ad.json

**Comment by SyntaxNode** [comment]
I agree this is confusing. The sample file being referenced is designed for integration tests and includes extra markup. I can see why it was originally linked to perhaps always stay current, but the samples don't change very often.

How about we just embed the sample request alongside the sample response? I can put out a PR for that tomorrow.

**Comment by SyntaxNode** [comment]
FYI: I'm fully aware the docs can use improvement. We're in the midst of a major overhaul, but I expect it to be a while before it's ready to be published. In the meantime, I ask for your patience and that you continue asking clarifying questions as GitHub issues.

**Comment by StarWindMoonCloud** [comment]
Thank you @SyntaxNode for the quick fix
